Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Debian builds the docs about 20% faster than alpine. The image is larger
+# and takes longer to build but that is worth it.
+FROM bitnami/minideb:stretch
```

Comment:
Alpine is much nicer to work with but 20% slower is a big deal given how often we run this locally.